src/main.rs

```diff
@@ -538,7 +538,8 @@ fn listen_ports<T: FromIterator<u16>>(conf: Config) -> T {
             | File::Stderr { .. }
             | File::Connect { .. } => None,
             File::Listen { port, prot, .. } => match prot {
-                Protocol::Tls | Protocol::Tcp => Some(port),
+                Protocol::Tls => Some((port, Some("https"))),
```

Comment:
Return either `https://<uri>` or `http://<url>` instead. The browser wouldn't really handle any other protocol and that's the only protocols we use